notebooks/book2/11/mcmc_traceplots_unigauss.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            return states, {"states": states, "info": info}
</pre><p>Not clear what <code>info</code> is?  Is this something your implementation adds or something provided out of the box by BlackJax?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/908/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, <code>info</code> is returned by blackjax model, I've added comment to clarify it. </p>